examples/demo_all.pp

```diff
@@ -31,10 +31,12 @@
   include ciscopuppet::demo_command_config
   include ciscopuppet::demo_domain
   include ciscopuppet::demo_interface
+  include ciscopuppet::demo_interface_portchannel
   #include ciscopuppet::demo_interface_service_vni
   include ciscopuppet::demo_ntp
   include ciscopuppet::demo_ospf
   include ciscopuppet::demo_patching
+  include ciscopuppet::demo_portchannel_global
```

Comment:
Also, please run this against unsupported platforms to validate the `undef` settings.